readme.md

```diff
@@ -179,25 +215,26 @@ Formless manages validation and failed submit attempts on your behalf, only noti
 -- A type representing only the successful parsed values in your Form type
 newtype OutputField error input output = OutputField output
 
--- `unwrapOutputs` is a helper function that will unwrap all these newtypes on your
--- behalf. Used on our custom Form type, it'd apply this transformation:
-unwrapOutputs' :: Form F.OutputField -> { name :: String, password1 :: Encrypted, password2 :: Encrypted }
-unwrapOutputs' = F.unwrapOutputs
+-- `unwrapOutput` is a helper function that will unwrap all these newtypes on your behalf.
+-- Used on our custom Form type, it'd apply this transformation:
+unwrapOutput' :: Form F.OutputField -> FormOutput
```

Comment:
`FormOutput` isn't actually a type that's been defined in this README. I think I'd like to keep it as the record, no type alias, to make it clear that the output is just the fields with their output values at the end, nothing fancier than that. What do you think?